src/Game/Util/GameSettings.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <Engine/Util/Settings.hpp>
+
+/// %Settings for the game.
+/**
+ * Available settings:
+ * Name                   | Description                      | Type   | Default Value
+ * ---------------------- | -------------------------------- | ------ | -------------
+ * Texture Reduction      | Reduction in texture quality.    | long   | 0
+ */
+class GameSettings : public Settings {
```

Comment:
Maybe this class should be organized in the `Utility` namespace?

Reply:
It's based on `EditorSettings` which is not in the namespace.